src/index.html

```diff
@@ -46,6 +46,12 @@
     <!-- <script src="https://cdn.rawgit.com/robflaherty/riveted/master/riveted.min.js"></script>
     <script>riveted.init();</script> -->
     <!-- End Google Tag Manager -->
+    <!-- Web socket connection -->
+    <script src="http://eb-ci.wmm63vqska.eu-west-1.elasticbeanstalk.com/primus/primus.js"></script>
```

Comment:
I was wondering if there was any way for us to "obfuscate" this URL to avoid it getting DDOSed ... but *realistically* there isn't a *simple* way...